### DISCUSSION TITLE
Production keys

### DISCUSSION THREAD

**Discussion by AbrarHayath** [skyline-emu/skyline#399] [Q&A] [answered]
Guys can plz someone say how to get the production keys?

**Comment by AndresNorm** [comment, marked as answer]
By dumping from your Switch

**Reply by AndresNorm** [reply]
You need a Switch to dump & play games on Emulator.
Emulator didn't build for Piracy purposes.

You should close this issue now.

**Reply by MKGAMES6** [reply]
But Where Can I Find The Title Keys And Product Keys Without A Switch?

**Reply by Chlorobite** [reply]
> But Where Can I Find The Title Keys And Product Keys Without A Switch?

You don't. Read the message you are replying to. You get them from a Switch.

**Reply by MKGAMES6** [reply]
I need the title key and the product key but "WITHOUT" A Switch If You Can't See The Reply Bro.

**Reply by iniMerchant64** [reply]
And you cant get keys without one. Go buy a switch

**Reply by MKGAMES6** [reply]
Why Then I Would Even Use The Emulator In The First Place😂

**Reply by iniMerchant64** [reply]
Good question. Emulator isnt a replacement to switch. In general to use ANY switch emu including yuzu and ryujinx you need to own the console.

**Reply by MKGAMES6** [reply]
Well then have fun creating a switch emulator (EVEN IF YOU TELL US TO BUY THE SWITCH FOR THE EMULATOR)

**Reply by iniMerchant64** [reply]
How entitled of you. Maybe you should take into account that in order for skyline to be legal we cannot provide any copyrighted content like the mentioned KEYS which can be aquired only from a previously mentioned console. At this point you really dont know how emulators function on the legal side. Have you ever seen a ps2 emu that comes with a bios? No? Then why do you expect a switch emu to come with keys?

**Reply by DamienStaebler** [reply]
emulator isnt a replacement for switch It's made for like streaming so you don't need capture card. and consoles like Wii U and Wii you can take laptop with you outside of house and play. So emulator isnt a way to avoid getting the real hardware

**Reply by iniMerchant64** [reply]
Not quite, its main goal is to extend the lifespan of a console. In general an emulator of a current gen console like Yuzu and Ryu allows you to play your games at a better quality or/and better framerates.

**Reply by Trublu72** [reply]
Can anyone please help me obtain the keys for pokemon legends arceus

**Reply by HalogenGam3r** [reply]
You need to have your switch to get the keys. Getting keys online is piracy.

**Reply by Trublu72** [reply]
I have the OG game but can't take switch to work

**Reply by HalogenGam3r** [reply]
Get a Hacked Switch if you have it.

**Reply by Trublu72** [reply]
Don't have a hacked switch YET but do have a v1 switch
